### PR TITLE
move functions from HCA helpers to platform

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
The `getMedicalCenterNameByID` function is going to be needed for the My VA Dashboard app so I'm moving it to `platform`. Also moved `medicalCenterLabels` to `platform` since it is used by `getMedicalCenterNameByID`.

## Testing done
Unit tests

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by sanlouise — APPROVED** [review]
Looks good, it seems we also still need to update the relative import paths in `src/applications/hca/config/form.js` to the helpers that are now in `platform`

File: src/applications/hca/helpers.jsx [hunk 1]
```diff
@@ -20,6 +20,11 @@ import { isInMPI } from 'platform/user/selectors';
 
 import facilityLocator from '../facility-locator/manifest.json';
 
+export {
```
Comment (erikphansen) [inline src/applications/hca/helpers.jsx:23]: @sanlouise anything that was importing the functions from this file will still work because of these lines.

**Review by sanlouise — APPROVED** [review]

File: src/applications/hca/helpers.jsx (hunk 1 above)
Comment (erikphansen) [reply src/applications/hca/helpers.jsx:23]: I actually don't know if this is a good approach or not. If reviewers object, I can find all the places that import the helpers from this file and update those imports.

**Review by rianfowler — CHANGES_REQUESTED** [review]
I think this PR should have a codeowners update, too. I don't think VSP FE should be the codeowner of these helpers- we don't have enough context to maintain them.

**Comment by erikphansen** [comment]
@rianfowler Who should own them? They are going to be used by the HCA and Auth Exp. The ability to easily look up a medical facility's name from its ID feels like a foundational/platform-y thing to me. 🤷‍♂️

**Review by rianfowler — APPROVED** [review]

**Pull request merged**